src/features/canvass/components/CanvassSidebar/index.tsx

```diff
@@ -13,12 +13,19 @@ import { ZetkinAreaAssignment } from '../../../areaAssignments/types';
 import ZUIMarkdown from 'zui/ZUIMarkdown';
 import { Msg } from 'core/i18n';
 import messageIds from 'features/canvass/l10n/messageIds';
+import useAreaAssignmentStats from 'features/areaAssignments/hooks/useAreaAssignmentStats';
+import ZUIFutures from 'zui/ZUIFutures';
 
 type Props = {
   assignment: ZetkinAreaAssignment;
 };
 
 const CanvassSidebar: FC<Props> = ({ assignment }) => {
+  const statsFuture = useAreaAssignmentStats(
+    assignment.organization_id,
+    assignment.id
+  );
+
```

Comment:
This loads the stats for the entire assignment. I think that for this issue, we really want to display just the stats for the current area.

That's harder to do, because it can't be loaded from the API. Instead, you have to use the locations that have been loaded and accumulate their stats.

There is a hook that used to be able to do this, called `useAssignmentAreaStats()`, but we had to disable it when we switched from the experimental API to the real API, because the real one did not support it.

Reply:
Confirmed fixed in 6257dd5ad2afd1148c9c2cb815528b5e92218a9b.